os/ex/devices/ST/lis2dw12.c

```diff
@@ -480,24 +497,11 @@ msg_t lis2dw12Start(LIS2DW12Driver *devp, const LIS2DW12Config *config) {
 
   /* Control register 6 configuration block.*/
   {
-    cr[6] = devp->config->accbadwidthselect |
+    cr[5] = devp->config->accbadwidthselect |

```

Comment:
@dron0gus did they fix this in newer versions or shall we at least report https://sourceforge.net/p/chibios/bugs/
?

Reply:
Nope, couse this is our driver.
```
commit ceb748118b6a84080ef18f79fd3f69ee698a4c41
Author: Andrey Gusakov <dron0gus@gmail.com>
Date:   Wed Dec 13 14:27:02 2023 +0300

    Devices: ST: add LIS2DW12 accelerometer
```